sdk/eventhub/Azure.ResourceManager.EventHubs/src/Generated/Models/EventHubsNetworkSecurityPerimeterConfiguration.cs

```diff
@@ -12,11 +12,10 @@
 namespace Azure.ResourceManager.EventHubs.Models
 {
     /// <summary> Network Security Perimeter related configurations of a given namespace. </summary>
-    public partial class EventHubsNetworkSecurityPerimeterConfiguration : TrackedResourceData
+    public partial class EventHubsNetworkSecurityPerimeterConfiguration : ResourceData
```

Comment:
This is a breaking change https://apiview.dev/Assemblies/Review/55488f1a7fb94d18a477b5c98fa608cf/81cff07e8f6f4371aa80f8e7233a17ad?diffRevisionId=78fe242374404a77944c32ac310ab8ac&doc=False&diffOnly=True#895f0ee6a5164dd6bf0b87fdea2c65cf